packages/types/src/config.ts

```diff
@@ -19,6 +19,7 @@ export enum ExtensionSettingType {
   Boolean = "boolean",
   Number = "number",
   String = "string",
+  MultilineTextInput = "multilinetextinput",
```

Comment:
This is way too long for my liking
```suggestion
  MultilineTextInput = "multiline",
```

Reply:
Also maybe it'd be nicer to call it `MultilineString` or something? It's weird for `String` to be called string and a multiline string suddenly being called a text input

Reply:
I think I agree with MultilineString for this, bit shorter than `multilinetextinput` but also more descriptive

alternatively I could MultilineTextInput -> MultilineString and `multilinetextinput` -> `multiline`

Reply:
I'd be fine with `multilinetextinput` -> `multilinestring`